analyzer/codechecker_analyzer/analyzers/clangsa/ctu_autodetection.py

```diff
@@ -11,12 +11,12 @@
 from __future__ import division
 from __future__ import absolute_import
 
-import os
-import re
 import subprocess
 
 from codechecker_common.logger import get_logger
 from codechecker_analyzer import host_check
+from codechecker_analyzer.analyzers.clangsa import version
```

Comment:
You can import these in one line:
```suggestion
from codechecker_analyzer.analyzers.clangsa import clang_options, version
```